add_users.py

```diff
@@ -0,0 +1,44 @@
+from app.db import get_session
+from app.models.users import Users
+from sqlmodel import select
+from app.auth.security import get_password_hash
+
+def add_users(users_list_dict: dict):
+    """
+    This function creates new users in the spun up local database
+
+    Args:
+        users_list_dict: Should contain a list of dictionary 
+        usernames and plain text passwords for users
+
+    Raises:
+        Print: If username already exists will print out warning
+        and ignore adding this user.
```

Comment:
Raises should only be used to list which exceptions may be raised when the function is called rather than side-effects.

It is worth nothing that it will log this in the DocString, however, can we please user `logger.warning()`/ `logger.info()` to log these sorts of messages. Which are great to have BTW! =D